api/v1alpha1/bundle_webhook.go

```diff
@@ -76,6 +76,14 @@ func checkBundleSource(r *Bundle) error {
 		if r.Spec.Source.Git == nil {
 			return fmt.Errorf("bundle.spec.source.git must be set for source type \"git\"")
 		}
+		if r.Spec.Source.Git.Directory != "" {
+			if r.Spec.Source.Git.Directory[0] == '/' {
+				return fmt.Errorf("bundle.spec.source.git.directory can be \"/\"")
+			}
```

Comment:
I think this should be allowed, and we should interpret it as the root of the repo. What would also mean that we'd treat `/foo` to mean that we will unpack `<repoRoot>/foo` as the bundle directory.

Reply:
I see.  I'll take it out.  Thanks.